### ISSUE TITLE
Could we please get better documentation?

### ISSUE THREAD

**Issue by kevinpostal** [irccloud/irccloud-python-client#1]
The Documentation for this project is lacking....

Also the RPC API docs are also incomplete.

https://github.com/irccloud/irccloud-tools/wiki/API-Overview

I see no way of getting current CID's for servers the user is already connected to.

**Comment by jwheare** [comment]
There is currently no way  to *just* get a list of cids, without fetching the full backlog via the stream. This is because we don't have a use for it internally, since we always use the stream.

We might change that in future, but until then, you can follow the instructions in the API overview for fetching `oob_include` and collect the cids from the `makeserver` messages.

**Comment by kevinpostal** [comment]
Is this supported using the HTTPS API ? I don't see any information for fetching oob_include

**Comment by jwheare** [comment]
Yes if you use the https://www.irccloud.com/chat/stream endpoint mentioned here: https://github.com/irccloud/irccloud-tools/wiki/API-Overview#https

**Comment by jwheare** [comment]
See also the https://github.com/irccloud/irccloud-tools/wiki/API-Overview#backlog-processing section.
